<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>智慧医疗心内科门诊系统问诊软件</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/imgs/consult.png</normaloff>:/icons/imgs/consult.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/imgs/start1.png</normaloff>:/icons/imgs/start1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Space</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/imgs/mute.png</normaloff>:/icons/imgs/mute.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="shortcut">
         <string>M</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actOpenDir"/>
    <addaction name="actAddDir"/>
    <addaction name="actDelDir"/>
    <addaction name="actTts"/>
    <addaction name="actDelFile"/>
    <addaction name="actTape"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="windowTitle">
    <string>音频目录树</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>音频目录树</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDir"/>
   <addaction name="separator"/>
   <addaction name="actRefresh"/>
   <addaction name="separator"/>
   <addaction name="actAddDir"/>
   <addaction name="separator"/>
   <addaction name="actDelDir"/>
   <addaction name="separator"/>
   <addaction name="actTts"/>
   <addaction name="separator"/>
   <addaction name="actTape"/>
   <addaction name="separator"/>
   <addaction name="actAddFile"/>
   <addaction name="separator"/>
   <addaction name="actDelFile"/>
  </widget>
  <action name="actAddDir">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/addfolder.png</normaloff>:/icons/imgs/addfolder.png</iconset>
   </property>
   <property name="text">
    <string>新建文件夹</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;新建文件夹&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actOpenDir">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/openfolder.png</normaloff>:/icons/imgs/openfolder.png</iconset>
   </property>
   <property name="text">
    <string>打开目录</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;打开文件夹&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actRefresh">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/refresh.png</normaloff>:/icons/imgs/refresh.png</iconset>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;刷新目录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actDelDir">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/delfolder.png</normaloff>:/icons/imgs/delfolder.png</iconset>
   </property>
   <property name="text">
    <string>删除目录</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;删除文件夹&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actTts">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/tts.png</normaloff>:/icons/imgs/tts.png</iconset>
   </property>
   <property name="text">
    <string>语音合成</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;语音合成&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actDelFile">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/delfile.png</normaloff>:/icons/imgs/delfile.png</iconset>
   </property>
   <property name="text">
    <string>删除文件</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;删除文件&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actTape">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/tape.png</normaloff>:/icons/imgs/tape.png</iconset>
   </property>
   <property name="text">
    <string>录音</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;录制音频&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actAddFile">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/imgs/addfile.png</normaloff>:/icons/imgs/addfile.png</iconset>
   </property>
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;添加文件&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
